### PR TITLE
Adds typing for wkhtmltopdf

### PR DESCRIPTION
**Pull request by digijap**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## wkhtmltopdf ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wkhtmltopdf@latest/))
was missing the following properties:
1. command
1. shell
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b36bae6272583d33da87bfbf5bd88a7f38b59420
</p>

File: types/wkhtmltopdf/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "esModuleInterop": true,
```
Comment (peterblazejewicz) [inline types/wkhtmltopdf/tsconfig.json:11]:
```suggestion
```
can you remove this? It's for TS consumption (to ease devex), not useful here in DT contexts, and this package is a plain, CJS module

File: types/wkhtmltopdf/wkhtmltopdf-tests.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import wkhtmltopdf from "wkhtmltopdf";
```
Comment (peterblazejewicz) [inline types/wkhtmltopdf/wkhtmltopdf-tests.ts:1]:
```suggestion
import wkhtmltopdf = require('wkhtmltopdf');
```
as per readme:
var wkhtmltopdf = require('wkhtmltopdf');

File: types/wkhtmltopdf/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+ *
+ * @param html HTML to convert to PDF
+ * @param options Options with the output parameter
+ */
+declare function wkhtmltopdf(html: string, options: OptionsOutfile): void;
+/**
+ * Call wkhtmltopdf and write PDF directly to specified file
+ *
+ * @param html HTML to convert to PDF
+ * @param options Options without the output parameter
```
Comment (peterblazejewicz) [inline types/wkhtmltopdf/index.d.ts:37]:
```suggestion
 * @param [options] Options without the output parameter
```

File: types/wkhtmltopdf/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+ */
+declare function wkhtmltopdf(html: string, options?: Options): NodeJS.ReadWriteStream;
+/**
+ * Call wkhtmltopdf and write PDF directly to specified file
+ *
+ * @param url URL to convert to PDF
+ * @param options Options without the output parameter
```
Comment (peterblazejewicz) [inline types/wkhtmltopdf/index.d.ts:44]:
```suggestion
 * @param [options] Options without the output parameter
```

File: types/wkhtmltopdf/wkhtmltopdf-tests.ts (hunk 2 above)
Comment (digijap) [reply types/wkhtmltopdf/wkhtmltopdf-tests.ts]: Thanks, I didn't know about this syntax. I've read about it and this makes way more sense :+1:

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@digijap thx!

**Comment by digijap** [comment]
Thank you for reviewing @peterblazejewicz !

**Pull request merged**